doc/papers.md

```diff
@@ -41,6 +46,8 @@ last_updated: September 9, 2015
 
 [Semi-automatic code-to-code transformer for Java: Transformation of library calls](http://liu.diva-portal.org/smash/get/diva2:944700/FULLTEXT01.pdf) (Boije, Niklas and Borg, Kristoffer ), Linköping University, 2016 
 
+[AeminiumGPU – A CPU-GPU Hybrid Runtime for the Aeminium Language](http://aeminium.dei.uc.pt/images/7/73/Alcides_fonseca_msc_thesis.pdf) (Fonseca, Alcides), University of Coimbra, 2011 
```

Comment:
Yes, I evaluated polyglot and spoon, and spoon was closer to what I wanted.

But the code I have from that time is not compatible at all with current spoon versions (https://github.com/AEminium/AeminiumGPUCompiler)